LDAR_Sim/src/constants/output_file_constants.py

```diff
@@ -18,11 +18,58 @@
 ------------------------------------------------------------------------------
 """
 
+from dataclasses import dataclass
+
 
 class FileDirectory:
     SUMMARY_PROGRAM_PLOTS_DIRECTORY = "program_summary_plots"
 
 
+class FileNames:
+    TRUE_VS_ESTIMATED_PERCENT_DIFF_PLOT = "True_vs_Estimated_Emissions_percent_differences"
+    TRUE_VS_ESTIMATED_RELATIVE_DIFF_PLOT = "True_vs_Estimated_Emissions_relative_differences"
+    TRUE_AND_ESTIMATED_PAIRED_EMISSIONS_DISTRIBUTION_PLOT = (
+        "True_and_Estimated_Paired_Emissions_Distribution"
+    )
+
+
+@dataclass
+class OutputConfigCategories:
+    PROGRAM_OUTPUTS = "Program Outputs"
+    SUMMARY_OUTPUTS = "Summary Outputs"
+    SUMMARY_VISUALIZATIONS = "Summary Visualizations"
+
+    @dataclass
+    class SummaryOutputCatageories:
+        SUMMARY_FILES = "Summary Files"
+        SUMMARY_STATS = "Summary Stats"
+
+
+@dataclass
+class SummaryFileNames:
```

Comment:
This would be better in the file_name_constants.py
In addition to this, the file extension should be specified here or the save_summary_file in summary_output_helpers.py should change to add the file extension. 
Currently the file output names the files Timeseries Summary or Emissions Summary without a file extension.

Reply:
TS_SUMMARY_FILENAME and EMIS_SUMMARY_FILENAME already exist in file_name_constants.py

Reply:
This is because all lookups are done without the file extensions and saving these with the file name extensions will cause extensions to be needed in the outputs parameters file or some processing to be done. In this case it makes more sense to simply add the file extensions afterwards.

Reply:
Will change save summary file to add the file extensions